world/worlddb.cpp

```diff
@@ -398,8 +447,27 @@ int WorldDatabase::MoveCharacterToBind(int CharID, uint8 bindnum)
 		heading = atof(row[5]);
 	}
 
-	query = StringFormat("UPDATE character_data SET zone_id = '%d', zone_instance = '%d', x = '%f', y = '%f', z = '%f', heading = '%f' WHERE id = %u",
-						 zone_id, instance_id, x, y, z, heading, CharID);
+	query = fmt::format(
```

Comment:
Using repositories would clean stuff like this up big time, not needed for this PR though